recipes/qcdloop/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "qcdloop" %}
+{% set version = "2.0.9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/scarrazza/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 6ffbd371658726054b903dce9a7855ac02b258eac59e2ee13524dda9e278f23b
+  folder: source
+  patches:
+    # Avoid egrep warning breaking qcdloop-config
+    - 0001-fix-Use-grep-E-over-egrep.patch
+
+build:
+  # FIXME: clang does not yet support quadmath, so macOS builds are broken
+  skip: true  # [not linux]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('qcdloop', max_pin='x.x') }}
+  script:
+    # FIXME: Remove unknown argument: '-fext-numeric-literals' CMAKE_CXX_FLAGS for macOS builds
+    # Note that conda-forge sed is gsed even on macOS
```

Comment:
As noted in https://github.com/scarrazza/qcdloop/issues/26#issuecomment-2469615057, `qcdloop` can be built on macOS, but it requires `g++` there instead of `clang`. I can figure out after this is merged how to force the macOS CI to use `g++`.